docs/reference/filters.md

```diff
@@ -1695,6 +1695,212 @@ oidcClaimsQuery("/:name%\"*One\"", "/path:groups.#[%\"*-Test-Users\"] groups.#[=
 
 As of now there is no negative/deny rule possible. The first matching path is evaluated against the defined query/queries and if positive, permitted.
 
+## Open Policy Agent
```

Comment:
I really like these docs, but I would put it into docs/tutorials/auth.md. The filter reference should explain the filter itself from the filter user pov not how to setup skipper. A link from reference to the tutorial will help to find the setup, in case someone found the filter reference and wants to understand how to set skipper up to do auth with OPA. I hope this makes sense to you.

Reply:
Totally, will split